swagger.yml

```diff
@@ -3,6 +3,25 @@ info:
   version: 0.11.4
   title: Netlify's API definition
   termsOfService: https://www.netlify.com/legal/terms-of-use/
+  description: >-
+    Netlify is a hosting service for the programmable web. It understands your
+    documents and provides an API to handle atomic deploys of websites, manage
+    form submissions, inject JavaScript snippets, and much more. This is a
+    REST-style API that uses JSON for serialization and OAuth 2 for
+    authentication.
+
+
+    This document describes common usage in some detail. There is also an
+    OpenAPI reference for the Netlify API that you can explore. Visit our
+    Community forum to join the conversation about understanding and using
```

Comment:
I'd also add back the community link that's in that intro at netlify.com/docs/api.

Reply:
Fixed in a5ad674022a55d0b4b3210f03cb5d4c4fac83ce0